_includes/devdoc/ref_p2p_networking.md

```diff
@@ -64,10 +63,11 @@ with the most recent versions listed first. (If you know of a protocol
 version that implemented a major change but which is not listed here,
 please [open an issue][docs issue].)
 
-As of Bitcoin Core 0.13.0, the most recent protocol version is 70014.
+As of Bitcoin Core 0.14.1, the most recent protocol version is 70015.
```

Comment:
I think a description of the 70015 changes should be included in the table below.